docs/documentation/development/bindings/thing-definition.md

```diff
@@ -249,12 +292,57 @@ Depending on the solution the provided meta data can be used for different purpo
     </thing-type>
 ```
 
-In general each `property` must have a name attribute which should be written in camel case syntax. The actual property value is defined as plain text and is placed as child node of the property element. It is recommended that at least the vendor and the model id properties are specified here since they should be definable for the most of the devices. In contrast to the properties defined in the 'ThingType' definitions the thing handler [documentation](thing-handler.html) explains how properties can be set during runtime.
+In general each `property` must have a name attribute which should be written in camel case syntax. 
+The actual property value is defined as plain text and is placed as child node of the property element. 
+It is recommended that at least the vendor and the model id properties are specified here since they should be definable for the most of the devices. 
+In contrast to the properties defined in the 'ThingType' definitions the thing handler [documentation](thing-handler.html) explains how properties can be set during runtime.
+
+### Representation Property
+
+A thing type can contain a so-called `representation property`. 
+This optional property contains the _name_ of a property, whose value can be used to uniquely identify a deivce. 
+The `thingUID` cannot be used for this purpose, there can be more than one thing for the same device. 
+
+Each physical device normally has some kind of technical id to identify it uniquely. 
+This could be a MAC address (e.g. Hue bridge, camera, all IP-based devices), a unique device id (e.g. a Hue lamp) or some other property that is unique per device. 
+This property is normally part of a discovery result for that specific thing type. 
+Having this property identified per binding it could be used as the `representation property` for this thing.
+
+The `representation property` will be defined in the thing type XML: 
+
+```xml 
+ <bridge-type id="thingTypeId">
+        ...
+        <properties>
+        	<property name="vendor">Philips</property>
```

Comment:
wouldn't it rather make sense to have the "serialNumber" property here? The "vendor" has no relation here...

Reply:
Well, this is a snippet from the real thing type XML. The thing type XML does NOT contain the properties, that are dynamically set in the binding itself. The "serialNumber" is the reference to this name. Until now, we did not define these "dynamic" properties in thing type XML. Not sure, if we should make this and it would affect a lot of bindings I'm afraid.